src/test/resources/features/04_employment.feature

```diff
@@ -1447,14 +1099,14 @@ Feature: Employment Journeys - Income Tax Submission
 
   Scenario: Individual user has prior submission data, updates accommodation to no, then does the full accommodation relocation flow
     When the user logs into the service with the following details
-      |Redirect url       |/2021/start                                          |
-      |Credential Strength|strong                                               |
-      |Confidence Level   |200                                                  |
-      |Affinity Group     |Individual                                           |
-      |Nino               |BB444444A                                            |
-      |Enrolment Key 1    |HMRC-MTD-IT                                          |
-      |Identifier Name 1  |MTDITID                                              |
-      |Identifier Value 1 |1234567890                                           |
+      | Redirect url        | /2021/start |
```

Comment:
This isn't something you've done, but on line 1120 i don't understand why it changes travel section to no, this is all about accommodation being no so i think lines 1120-1124 can be removed

Reply:
Removed as suggested